### ISSUE TITLE
Local Only Communities

### ISSUE THREAD

**Issue by Pdzly** [sublinks/sublinks-api#342]

**Comment by gaviscapes** [comment]
Having a "locked down" community idea would be helpful where it's not only local only but only visible to logged in users as well

**Comment by jgrim** [comment]
> Having a "locked down" community idea would be helpful where it's not only local only but only visible to logged in users as well

This seems like a great idea. Could you create it as a separate issue? This could be useful for all community types.

**Comment by jgrim** [comment]
I don't understand the need for local only communities. Doesn't that kinda break the idea of federation?

**Comment by DraconicNEO** [comment]
> I don't understand the need for local only communities. Doesn't that kinda break the idea of federation?

I agree, it kind of goes against the whole idea of federation. Local only posts on Mastodon make sense where you're speaking as yourself and might only want to get feedback from your instance on something, but not for communities in a forum. Restricting communities so they're for a specific group maybe but not private or local only. For those it's better to use a different platform.

**Comment by gaviscapes** [comment]
> I don't understand the need for local only communities. Doesn't that kinda break the idea of federation?

It's important for vulnerable communities and instances who want to post public moderation notices that aren't publicly swamped with brigading from other instances. No matter how hard you ask for others not to post their opinions if it doesn't apply to them, they will do it anyway. As well in some cases it helps establish more of a local instance culture. It's good to have the option at least.

**Comment by gaviscapes** [comment]
> > I don't understand the need for local only communities. Doesn't that kinda break the idea of federation?
> 
> I agree, it kind of goes against the whole idea of federation. Local only posts on Mastodon make sense where you're speaking as yourself and might only want to get feedback from your instance on something, but not for communities in a forum. Restricting communities so they're for a specific group maybe but not private or local only. For those it's better to use a different platform.

The purpose is that it's there for those who want it, the granularity is the point. As well see above my point too in response to this. For some it's a matter of safety. It should also be noted this is a feature lemmy is working on, and not implementing it would break API parity.

**Comment by Pdzly** [comment]
I agree to @gaviscapes, just to have the option is better than not to have it.

For vulnerable communities it is benefitial or communities that you dont want to have to worry about federated reports or just want to have a local community ( For better controll over your data ).

I am really rooting for this feature.

**Comment by jgrim** [comment]
I understand, thanks for the clarity.
